templates/home/join_us.html.twig

```diff
@@ -11,53 +11,53 @@
-                                    Dołącz do naszego <a href="{{ url.discord }}">Discorda</a> i złóż podanie na kanale #rekrutacja
-                                </li>
-                                <li>
-                                    Gdy już otrzymasz rangę rekruta będziesz mógł zalogować się na stronie i dołączyć do naszego
-                                    {%- if is_granted('ROLE_USER') -%}
-                                        <a href="{{ url.teamspeak }}">TeamSpeaka</a>
-                                    {%- else %}
-                                        <a href="#" title="{{ 'Available after login'|trans }}"
-                                           data-toggle="tooltip" data-placement="bottom">TeamSpeaka</a>
-                                    {%- endif -%}
-                                </li>
-                                <li>
-                                    Pobierz mody z modlisty "<a href="https://modlist.armaforces.com/#/download/default">default</a>". Są one wymagane na większości misji
-                                    <br/>
-                                    <br/>
-                                    Standardowo rozgrywki odbywają się w środy oraz soboty o godzinie 20:00, po rozegraniu swojej pierwszej misji
-                                    nie będziesz już rekrutem i zostaniesz przyjęty do grupy.
-                                </li>
-                            </ul>
-                        </section>
-                        <section class="text-left">
-                            <h5>{{ 'What do we require to from recruits'|trans }}</h5>
-                            <ul>
-                                <li>{{ 'Base Arma 3 game'|trans }}</li>
-                                <li>{{ 'At least 16 years old'|trans }}</li>
-                            </ul>
-                        </section>
+                <div class="row d-flex justify-content-center text-center">
+                    <!-- Card -->
+                    <div class="card bg-dark my-4 mx-auto col-md-12 col-lg-6">
+                        <div class="card-body">
+                            <h4 class="card-title">{{ 'Join ArmaForces'|trans }}</h4>
+                            <section class="text-left">
+                                <ul id="join-steps">
+                                    <li>
+                                        Zacznij od zapoznania się z <a href="{{ url.statute }}">regulaminem</a>
+                                    </li>
+                                    <li>
+                                        Dołącz do naszego <a href="{{ url.discord }}">Discorda</a> i złóż podanie na kanale #rekrutacja
+                                    </li>
+                                    <li>
+                                        Gdy już otrzymasz rangę rekruta będziesz mógł zalogować się na stronie i dołączyć do naszego
+                                        {%- if is_granted('ROLE_USER') -%}
```

Comment:
This white space should not be stripped as this results in no space between words.
```suggestion
                                        {%- if is_granted('ROLE_USER') %}
```

Reply:
Fixed